app/main.py

```diff
@@ -2,24 +2,78 @@
 
 
 class IntegerRange:
-    pass
+    def __init__(self, min_amount: int, max_amount: int) -> None:
+        self.min_amount = min_amount
+        self.max_amount = max_amount
+
+    def __set_name__(self, instance: None, name: str) -> None:
```

Comment:
add typehints
https://stackoverflow.com/questions/3798835/understanding-get-and-set-and-python-descriptors